### PR TITLE
Release Flux v1.12.3

### PR DESCRIPTION
**Pull request by 2opremio**

### CODE REVIEW

**Comment by hiddeco** [comment]
2c832467d1aeb6d4a8b65573a7fb6e2783b79a94 originates from #1791 but is probably included because of 6466e32fa3647d012d7b239f01b0d29116657001?

**Comment by 2opremio** [comment]
> [2c83246](https://github.com/weaveworks/flux/commit/2c832467d1aeb6d4a8b65573a7fb6e2783b79a94) originates from #1791 but is probably included because of [6466e32](https://github.com/weaveworks/flux/commit/6466e32fa3647d012d7b239f01b0d29116657001)?

Yep

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,46 @@
 This is the changelog for the Flux daemon; the changelog for the Helm
 operator is in [./CHANGELOG-helmop.md](./CHANGELOG-helmop.md).
 
+## 1.12.3 (2019-05-22)
+
+This is a patch release.
+
+### Fixes
+
+- Show tag image for workload in list-images
```
Comment (squaremo) [inline CHANGELOG.md]: This is the title of the PR but it's not self-explanatory. I'd suggest "Show untagged images in fluxctl list-images"

Comment (2opremio) [reply CHANGELOG.md]: Done

**Pull request merged**